Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'berkshelf', '~> 5.1', '>= 5.1.0'
 
 group :unit do
   gem 'foodcritic',       '~> 8.1', '>= 8.1.0'
-  gem 'rubocop',          '~> 0.45', '>= 0.45.0'
+  gem 'cookstyle'
```

Comment:
I don't understand the motivation for cookstyle, why not just add additional rules to rubocop?